inst/components/scss/sidebar.scss

```diff
@@ -236,89 +269,75 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
 
 @include media-breakpoint-down(sm) {
   .bslib-sidebar-layout, .bslib-sidebar-layout.sidebar-right {
```

Comment:
Kind seems like you could get rid of this and move the CSS below it into the `.bslib-sidebar-layout {}` below it? 

```suggestion
```